WhizzML.sublime-completions

```diff
@@ -133,6 +134,7 @@
         {"trigger": "lambda", "contents": "lambda"},
         {"trigger": "last\tGet last element of a list", "contents": "last"},
         {"trigger": "let", "contents": "let"},
+        {"trigger": "levenshtein\tCompute (non-neg integer) the distance between two string values", "contents": "levenshtein $1 $2"},
```

Comment:
nit: i'd put the `(non-neg integer)` at the end, and use non-negative (unless then the string is too long).